incubator/sparkoperator/values.yaml

```diff
@@ -0,0 +1,5 @@
+operatorImageName: gcr.io/spark-operator/spark-operator
+operatorVersion: v2.3.0-v1alpha1-latest
+operatorNamespace: sparkoperator
+
+enableWebhook: false
```

Comment:
Should this be `true` instead?

Reply:
I think it should be `true` too. But one concern is that when it's `true`, the chart assumes that the user has already created the `sparkoperator` namespace, as documented in the README. If the user is not careful reviewing the README, he'll get an error when installing the chart. If you don't think it's an issue to worry about, I'm fine changing it to `true`.

Reply:
Makes sense, let's keep `false` as the default.

Reply:
Cool. Thanks for the review. Now comments are all addressed. Feel free to merge or ping the right person to merge.